test/code/plugins/oms_common_test.rb

```diff
@@ -94,6 +100,256 @@ def teardown
       "OSAlias=UniversalR\n" \
       "OSManufacturer=Central Logistics GmbH\n"
 
+    # Begin:  Unit Tests for some simple methods.
+    # May best be put in another file later.
+
+    def test_clean_hostname_string
+      hn = ' www.five.com:333     '
+      b = Common.clean_hostname_string(hn)
+      expected = 'www.five.com'
+      assert_equal(b,expected,"method clean_hostname_string should return '#{expected}'.")
+      hn = '        microsoft     '
+      b = Common.clean_hostname_string(hn)
+      expected = 'microsoft'
+      assert_equal(b,expected,"method clean_hostname_string should return '#{expected}'.")
+      hn = expected
+      b = Common.clean_hostname_string(hn)
+      assert_equal(b,expected,"method clean_hostname_string should return '#{expected}'.")
+    end
+
+    def test_has_Designated_HostnameFile?
+      file_hostname = '/tmp/containerhostname'
+      test_hostname = 'test_hostname'
+      Common.HostnameFilePath = file_hostname
+      out_file = File.new(file_hostname, "w+")
+      out_file.puts(test_hostname)
+      out_file.close
+      r = Common.has_Designated_HostnameFile?
+      assert(r,"method has_Designated_HostnameFile? should see #{file_hostname} exists at this point.")
+      File.delete(file_hostname)
+      r = Common.has_Designated_HostnameFile?
+      assert(!r,"method has_Designated_HostnameFile? should see #{file_hostname} Does NOT exist at this point.")
+      Common.HostnameFilePath = nil
+    end
+
+    def test_is_dot_separated_string?
+      s = "one.two"
+      r = Common.is_dot_separated_string?(s)
+      assert(r,"Common.is_dot_separated_string?(#{s}) should be true.")
+      s = "one.two.three.four"
+      r = Common.is_dot_separated_string?(s)
+      assert(r,"Common.is_dot_separated_string?(#{s}) should be true.")
+      s = "1.2.3.4"
+      r = Common.is_dot_separated_string?(s)
+      assert(r,"Common.is_dot_separated_string?(#{s}) should be true.")
+      s = "192.168.223.222"
+      r = Common.is_dot_separated_string?(s)
+      assert(r,"Common.is_dot_separated_string?(#{s}) should be true.")
+      s = "onetwo"
+      r = Common.is_dot_separated_string?(s)
+      assert(!r,"Common.is_dot_separated_string?(#{s}) should be true.")
+      s = ".one:two:three:four"
+      r = Common.is_dot_separated_string?(s)
+      assert(!r,"Common.is_dot_separated_string?(#{s}) should be true.")
+      s = "1234"
+      r = Common.is_dot_separated_string?(s)
+      assert(!r,"Common.is_dot_separated_string?(#{s}) should be true.")
+      s = "192:168:223:222."
+      r = Common.is_dot_separated_string?(s)
+      assert(!r,"Common.is_dot_separated_string?(#{s}) should be true.")
```

Comment:
"192:168:223:222." is not actually a dot-separated string, since it has no content after the dot. If you want to assert something is false, it is much clearer to use `assert_false` and correlate the message.

Reply:
Good point about assert false.  I'm used to the shoulda framework.